src/bootstrap/setup.rs

```diff
@@ -89,7 +89,7 @@ pub fn setup(src_path: &Path, profile: Profile) {
         std::process::exit(1);
     }
 
-    let path = cfg_file.unwrap_or_else(|| src_path.join("config.toml"));
+    let path = cfg_file.unwrap_or("config.toml".into());
```

Comment:
```suggestion
    let path = cfg_file.unwrap_or_else(|| "config.toml".into());
```

Reply:
Performance doesn't matter for x.py. I'd rather have it be easier to read.